how-to/customize-workspace/client/src/framework/platform/interopbroker.ts

```diff
@@ -1097,6 +1118,18 @@ export function interopOverride(
 			}
 			return app.appId;
 		}
+
+		private async processContext(context: OpenFin.Context): Promise<OpenFin.Context> {
+			const endpointId = `interopbroker.process.${context.type}`;
+			if (hasEndpoint(endpointId)) {
+				logger.info(`Processing context ${context.type} with endpoint ${endpointId}`);
+				const processedContext = await requestResponse<ContextToProcess, ProcessedContext>(endpointId, {
+					context
+				});
+				return processedContext.context;
```

Comment:
Would it be useful to log the original and processedContext here

Reply:
Depends on the sensitivity of the context object. Maybe as an option for the endpoint?